sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignature.java

```diff
@@ -556,6 +573,30 @@ public static TimerParameter timerParameter(TimerDeclaration decl) {
       TimeDomainParameter() {}
     }
 
+    /** Descriptor for a {@link Parameter} of type {@link DoFn.SideInput}. */
+    @AutoValue
+    public abstract static class SideInputParameter extends Parameter {
+      SideInputParameter() {}
+
+      public abstract TypeDescriptor<?> elementT();
+
+      @Nullable
+      public abstract String fieldAccessString();
```

Comment:
shouldn't be called fieldAccessString. Maybe sideInputId? Also shouldn't be nullable.

Reply:
Done